src/utils/Fetcher.ts

```diff
@@ -24,7 +23,19 @@ export class Fetcher {
     this.customDefaults = mergeRequestOptions(this.customDefaults, overrideDefaults)
   }
 
-  fetchJson(url: string, options?: RequestOptions): Promise<any> {
+  fetch(url: string, options: Partial<CompleteRequestOptions>): Promise<Response> {
```

Comment:
who will use this method? Is the idea to migrate from `fetchJson` to this? If that's the case, shouldn't we remove the other?

Reply:
I would encourage using `fetch` instead of `fetchJson`. But I don't want to generate a hard dependency and break every implemented code on top of it. Should I add a `// @deprecated ` notice to start discouraging it?

Reply:
Yes, please